packages/suite/src/views/dashboard/PortfolioCard/PortfolioCardException.tsx

```diff
@@ -121,9 +125,12 @@ const discoveryFailedMessage = (discovery?: Discovery) => {
         if (networkError.includes(account.symbol)) return value;
         networkError.push(account.symbol);
 
+        const accountTypeDisplay = account.accountType !== 'normal' ? account.accountType : '';
+
         return value.concat(
             <div key={account.symbol}>
-                {network.name}: {getAccountError(account.error)}
+                {network.name} {accountTypeDisplay}:{' '}
```

Comment:
This means, that if it is normal, there is ` `

Reply:
Removed, now the space will be only when the type is not normal